data/json/scenarios.json

```diff
@@ -358,15 +364,76 @@
   {
     "type": "scenario",
     "id": "lab_cargo_staff_1",
-    "name": "Challenge - Trans-Coast Logistics Concourse",
+    "name": "Challenge - Trans-Coast Logistics Staff",
     "points": -6,
     "description": "You were deemed an essential employee and required to stay behind during the facility's evacuation.  You better find your misplaced lanyard with your badge before the automated defenses notice that you don't have one.",
     "start_name": "Locked Lab",
-    "professions": [ "security" ],
+    "professions": [ "security", "labtech", "medic" ],
     "allowed_locs": [ "sloc_lab_concourse_area" ],
     "map_extra": "mx_lab_concourse_area",
     "flags": [ "CHALLENGE", "LONE_START" ]
   },
+  {
+    "type": "scenario",
+    "id": "lab_cargo_mutant_1",
+    "name": "Challenge - Trans-Coast Logistics Subject",
+    "points": -6,
+    "description": "You were confined in this facility and used to test mutagenic drugs until all hell broke loose.  You're lucky you still have your sanity, unlike many of the other subjects, because it's going to take all your cunning to get out of here alive.",
+    "start_name": "Locked Lab",
+    "professions": [ "mutant_patient", "mutant_volunteer", "convict_political" ],
+    "allowed_locs": [ "sloc_lab_concourse_area" ],
+    "map_extra": "mx_lab_concourse_area",
+    "traits": [
+      "ELFAEYES",
+      "URSINE_EYE",
+      "FANGS",
+      "GILLS",
+      "SPINES",
+      "PLANTSKIN",
```

Comment:
How much armor does this provide?

Reply:
https://nornagon.github.io/cdda-guide/#/item/integrated_plantskin
Phelloderm is currently *garbage*, having 5 encumbrance for 2.5 defense across the board, occupies PERSONAL. It also is completely non-breathable. It sucks so much ass that I can't help but feel it was a mistake.
https://nornagon.github.io/cdda-guide/#/item/integrated_lightfur
Lightfur is a lot better, at 2 encumbrance providing 4 bash, 2.5 cut, and 2 ballistic, though it's pretty warm, and barely breathable.
https://nornagon.github.io/cdda-guide/#/item/integrated_chitin
Epicuticle is currently broken as hell. For 1 encumbrance, you get perfect breathability, 7.5 bash, 5.0 cut, and 2.5 ballistic. It needs nerfed hard. But I don't know if that should make it non addable, it just needs to be balanced.

Reply:
https://github.com/CleverRaven/Cataclysm-DDA/pull/60505
This is what scales will look like, they might be a bit on the pushed side but unlike the other three the entry level scales do not have full coverage. Open to balancing considerations for those ones.